pkg/controllers/provisioningv2/rke2/bootstrap/controller_test.go

```diff
@@ -4,11 +4,15 @@ import (
 	"crypto/sha256"
 	"encoding/base64"
 	"fmt"
+
+	provisioningcontrollers "github.com/rancher/rancher/pkg/generated/controllers/provisioning.cattle.io/v1"
```

Comment:
```suggestion
	provcontrollers "github.com/rancher/rancher/pkg/generated/controllers/provisioning.cattle.io/v1"
```

This should probably stay `provcontrollers` so it's clear which import is for the controllers and which is for the types.